src/ctapipe/image/cleaning.py

```diff
@@ -459,6 +494,137 @@ def time_constrained_clean(
+        set to true.
+    4) Apply bright_cleaning - `ctapipe.image.cleaning.bright_cleaning` if
+        `apply_bright_cleaning` is set to true.
+    5) Get only largest island - `ctapipe.image.morphology.largest_island` if
+        `largest_island_only` is set to true.
+
+    Parameters
+    ----------
+    geom: `ctapipe.instrument.CameraGeometry`
+        Camera geometry information
+    image: `np.ndarray`
+        pixel charges
+    arrival_times: `np.ndarray`
+        Pixel timing information
+    picture_thresh: `float` or `np.ndarray`
+        threshold above which all pixels are retained. Used for `tailcuts_clean`.
+    boundary_thresh: `float` or `np.ndarray`
+        threshold above which pixels are retained if they have a neighbor
+        already above the picture_thresh. Used for `tailcuts_clean`.
+    min_number_picture_neighbors: `int`
+        A picture pixel survives cleaning only if it has at least this number
+        of picture neighbors. This has no effect in case keep_isolated_pixels is True.
+        Used for `tailcuts_clean`.
+    keep_isolated_pixels: `bool`
+        If True, pixels above the picture threshold will be included always,
+        if not they are only included if a neighbor is in the picture or
+        boundary. Used for `tailcuts_clean`.
+    apply_time_cleaning: `bool`
+        Set to true if `time_delta_cleaning` should be applied.
+    time_limit: `float`
+        Time limit for the `time_delta_cleaning`.
+    time_num_neighbors: int
+        Used for `time_delta_cleaning`.
+        A selected pixel needs at least this number of (already selected) neighbors
+        that arrived within a given time_limit to itself to survive this cleaning.
+    apply_bright_cleaning: `bool`
+        Set to true if `bright_cleaning` should be applied.
+    bright_cleaning_fraction: `float`
+        `fraction` parameter for `bright_cleaning`. Pixels below
+        fraction * (average charge in the 3 brightest pixels) will be removed from
+        the cleaned image.
+    bright_cleaning_threshold: `float`
+        `threshold` parameter for `bright_cleaning`. Minimum average charge
+        in the 3 brightest pixels to apply the cleaning.
+    largest_island_only: `bool`
+        Set to true to get only largest island.
+    apply_pedestal_cleaning: `bool`
+        Set to true if pedestal cleaning should be applied.
+    pedestal_factor: `float`
+        Factor for interleaved pedestal cleaning. It is multiplied by the
+        pedestal standard deviation for each pixel to calculate pixelwise picture
+        threshold parameters for `tailcuts_clean` considering the current background.
+    pedestal_std: `np.ndarray`
+        Pedestal standard deviation for each pixel. See
+        `ctapipe.containers.PedestalContainer`
+
+    Returns
+    -------
+    A boolean mask of *clean* pixels.
+
+    """
+    # Step 1
+    if apply_pedestal_cleaning and pedestal_std is not None:
```

Comment:
Why do we need two arguments if `None` for `pedestal_std` is used to activate/deactivate this?